kiva/pdf.py

```diff
@@ -602,7 +602,7 @@ def draw_image(self, img, rect=None):
             return
 
         if rect is None:
-            rect = (0, 0, img.width(), img.height())
+            rect = (0, 0, pil_img.width, pil_img.height)
```

Comment:
Sigh. I should've caught this in the earlier PR.

Reply:
No worries. That's what test coverage is for.